src/main/java/org/alliancegenome/curation_api/base/BaseSQLDAO.java

```diff
@@ -84,6 +96,13 @@ public E remove(String id) {
         return entity;
     }
 
+    public E remove(Long id) {
+        log.debug("SqliteDAO: remove: " + id);
+        E entity = find(id);
+        entityManager.remove(entity);
+        return entity;
+    }
+
```

Comment:
@cmpich good catch I didn't update these after we made the GeneratedCurie base class